trace-archiver/default.nix

```diff
@@ -43,6 +43,7 @@
       Env = [
         "SSL_CERT_FILE=${pkgs.cacert}/etc/ssl/certs/ca-bundle.crt"
         "OBSERVABILITY_ADDRESS=0.0.0.0:9090"
+        "METRICS_ADDRESS=0.0.0.0:9091"
```

Comment:
What is this for?

Reply:
I think because I had added [this line](https://github.com/STFC-ICD-Research-and-Design/supermusr-data-pipeline/blob/204ca8634b0dfbacee5392ad66405d68f128281f/trace-telemetry-exporter/default.nix#L39) previously, I assumed that the `METRICS_ADDRESS` and `OBSERVABILITY_ADDRESS` were meant to be different entities. What exactly is the `OBERVABILITY_ADDRESS` port used for? I thought that `9090` had something to do with the Kafka brokers?

Reply:
No, here the use of OpenMetrics is to monitor the application, commonly referred to as observability. In the telemetry exporter the tool itself is something that facilitates metrics being available.

In short, the correct action here is to use the observability address (as it previously worked).